docs/components/power-sensor/fake.md

```diff
@@ -19,10 +19,10 @@ Configure a `fake` power sensor to integrate into your machine:
 {{< tabs name="Configure a Fake Power Sensor" >}}
 {{% tab name="Config Builder" %}}
 
-Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com).
-Click on the **Components** subtab and click **Create component** in the lower-left corner.
-Select the type `power_sensor`, then select the `fake` model.
-Name your sensor, and click **Create**.
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+Select the `power_sensor` type, then select the `fake` model.
+Enter a name for your sensor and click **Create**.
```

Comment:
supernit: let's match the other pages where the type of sensor is also explicitly referenced:

```suggestion
Enter a name for your power sensor and click **Create**.
```

and on other power sensor pages.